### ISSUE TITLE
Division by zero in Vincenty.php on line 86

### ISSUE THREAD

**Issue by maffpool** [mjaschen/phpgeo#18]
Hello,

Thanks for this great library !

I'm facing this issue with version 1.3.5 and php cli version 5.5.9-1ubuntu4.14

```php
$vincenty = new \Location\Distance\Vincenty();

$distance = $vincenty->getDistance(
    new \Location\Coordinate(0, 0),
    new \Location\Coordinate(0, 0.1)
);
```

> PHP Warning:  Division by zero in /home/.../vendor/mjaschen/phpgeo/src/Location/Distance/Vincenty.php on line 86

Any idea ?

**Comment by mjaschen** [comment]
Hi,

I'm able to reproduce the error and will start to work on a fix.

Marcus

**Comment by mjaschen** [comment]
Ok, it's fixed now (version 1.3.6).

Thanks for reporting this bug!

**Comment by maffpool** [comment]
Great job, thanks
